packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -76,15 +85,47 @@ export interface EditableTree {
     readonly [proxyTargetSymbol]: object;
 
     /**
-     * Anchor to this node.
-     * Valid as long as this EditableTree's context is not freed.
-     * Might not point to any node if this node is deleted from the document.
-     *
-     * TODO: When a proper editing API is exposed on EditableTree directly,
-     * this should become an implementation detail and rbe removed from this API surface.
+     * Creates a child of this node.
+     */
+    readonly [insertNodeSymbol]: (key: string, value: ITreeCursor) => boolean;
+
+    /**
+     * Sets value of a child of this node.
+     */
+    readonly [setValueSymbol]: (key: string, value: unknown, typeName: TreeSchemaIdentifier) => boolean;
+
+    /**
+     * Deletes a child of this node.
      */
-    readonly [anchorSymbol]: Anchor;
+    readonly [deleteNodeSymbol]: (key: string) => boolean;
+}
 
+/**
+ * A tree where the forest has no data neither an initialized root node.
```

Comment:
The reason for existence of this type is unclear, and it seems to complicate the APIs.

Is this whats returned when accessing an empty optional field? If so, we should document that. Docs make it sound like its only used when the root is missing? If so we shouldn't introduce a whole new type for that: it adds unnecessary complexity.

Also this is documented in terms of "forest": EditiableTree is supposed to abstract over the forest, and should not require the user of it to understand forest (or even know it exists) to understand its documentation.